files/besu/smart_contracts/scripts/concurrentPrivateTxs.js

```diff
@@ -0,0 +1,130 @@
+const Web3 = require("web3");
```

Comment:
we should add this file to the `package.json` as a specific task like `concurent-private-tx` so users can do `npm run concurent-private-tx`